vm/gvisor/gvisor.go

```diff
@@ -332,6 +355,11 @@ func (inst *instance) Diagnose() ([]byte, bool) {
 	if err != nil {
 		b = append(b, []byte(fmt.Sprintf("\n\nError collecting stacks: %v", err))...)
 	}
+	b1, err := osutil.Run(time.Minute, osutil.Command("dmesg"))
```

Comment:
`osutil.RunCmd(time.Minute, "", "dmesg")` is a bit shorter and we tend to use RunCmd if nothing special needs to be done with the Command object.